index.html

```diff
@@ -10,224 +10,293 @@
+          fill="#505F79" />
+      </svg>
+    </a><br></br>
+  </div>
+  <br></br>
+  <div>
+    <button id="bttn">Get My Resume</button>
+  </div><br>
+
+  <ul>
+    <div class="fr">
+      <p class="la">Languages </p>
+      <img class="dropdown" src="images/drop.jpeg" alt="">
     </div><br>
+
+    <li class="li">
+      <img src="shapes/Ellipse 1.png" alt="">
+      <p id="java">Javascript</p>
+    </li><br>
+
+    <li class="li">
+      <img src="shapes/eclipse/Ellipse 1.png" alt="">
+      <p id="htm">HTML</p>
+    </li><br>
+
+    <li class="li">
+      <img src="shapes/eclipse 2/Ellipse 1.png" alt="">
+      <P id="cs">CSS</P>
+    </li><br>
+
+  </ul>
+
+  <div > 
     <ul>
-      <div class="fr">
-        <p class="la">Languages </p>
-        <img class="dropdown" src="images/drop.jpeg" alt="">
-      </div><br>
-      <li class="li">
-        <img src="shapes/Ellipse 1.png" alt="">
-        <p id="java">Javascript</p>
-      </li><br>
-      <li class="li">
-        <img src="shapes/eclipse/Ellipse 1.png" alt="">
-        <p id="htm">HTML</p>
-      </li><br>
-      <li class="li">
-        <img src="shapes/eclipse 2/Ellipse 1.png" alt="">
-        <P id="cs">CSS</P>
-      </li><br>
+      <li class="fr">
+        <h1>
+          <p class="la">Frameworks </p>
+        </h1>
+        <img class="dropdown" src="images/dropdown.jpeg" alt="">
+      </li>
+      <hr />
+      <li class="fr">
+        <h1>
+          <p class="la">Skills</p>
+        </h1>
+        <img class="dropdown" src="images/dropdown.jpeg" alt="">
+      </li>
+      <hr />
     </ul>
-    <div>
-      <ul>
-        <li class="fr">
-          <h1>
-            <p class="la">Frameworks </p>
-          </h1>
-          <img class="dropdown" src="images/dropdown.jpeg" alt="">
-        </li>
-        <hr />
-        <li class="fr">
-          <h1>
-            <p class="la">Skills</p>
-          </h1>
-          <img class="dropdown" src="images/dropdown.jpeg" alt="">
-        </li>
-        <hr />
-      </ul>
-    </div>
-  </section>
+  </div>
+</section>
+
+<section class="contact">
+
+  <div>
+   <h2 class="contact-text">Contact me</h2>
+   <div class="par-container"><p class="par-contact">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it</p></div>
+  </div> 
+  <div>
+   <form action="https://formspree.io/f/xeqwozwl" method="post">
+    <ul>
+     <li>
+       <input type="text" class="name" name="user_name" placeholder="Yeremias" required maxlength="30">
+     </li>
+
+     <li>
+       <input type="email" class="mail" name="user_email" placeholder="natajayanj14@gmail.com" required>
+     </li>
+
+     <li>
+      <textarea name="user_message" class="msg" cols="" rows="5" placeholder="Write your message here" required maxlength="500"></textarea>
+     </li>
```

Comment:
- [ ] Kindly reduce the padding of the inputs. In addition, ensure that the `textarea` input has a width that is equal to other inputs.

Your design 😎 
![contact](https://user-images.githubusercontent.com/99394374/213879655-02ed6702-982b-4e6a-81fb-bffc5f62bf37.PNG)